### PR TITLE
Update IO one by one

### PR DESCRIPTION
**Pull request by RisingOrange**

### CODE REVIEW

**Comment by AnKingMed** [comment]
Only issue I'm having now is that the Hint Hotkeys add-on won't open the buttons anymore. I think it may be because that worked with this line: `<a href="#"`. Not 100% sure though

I also wonder if we should change `Hint Settings` and `ToggleAllHints` to use `Button` instead of `Hint`. Thoughts on that?

**Comment by AnKingMed** [comment]
Also, mobile is looking weird. I can no longer tap the occlusions directly (which would be nice). The personal notes button always shows up white like this and the reveal buttons look different on mobile

IMG_4844

**Comment by AnKingMed** [comment]
Yeah lets make the buttons look the same. They can look slightly different than the other buttons (lecture note, etc), but should look the same on mobile and desktop.

**Comment by AnKingMed** [comment]
Few issues now:
1. My hint hotkeys still isn't working
2. The buttons expand to a larger size now instead of staying the same size on opening like they previously did
image

Maybe both of these would work if we revert to the older button style - is there a reason to do the new one other than it's cleaner?

3. My personal notes button still highlights by default on mobile
4. The reveal next and toggle all buttons still look different on mobile
5. I still can't tap the buttons to reveal them on mobile

File: Note Types/IO-one by one/Back Template.html [hunk 1]
```diff
@@ -27,48 +32,47 @@
 <div id="extras" hidden>{{I0}}</div>
 <div class="header">{{Header}}</div>
 <div class="io">{{Image}}</div>
+<button id="reveal-next-btn" class="button">Reveal Next</button>
+<button id="toggle-all-btn" class="button">Toggle All</button>
+<br>
 
 <!-- EXTRA FIELD -->
-{{#Extra}}<a href="#" class="hint" id="hint-ar" onclick="toggleHint('button-ar', 'hint-ar')">
```
Comment (AnKingMed) [inline Note Types/IO-one by one/Back Template.html:32]: Does this need to have `class="hint"` as well to make hint hotkeys work?

**Comment by AnKingMed** [comment]
I think I solved all of the issues in this branch: https://github.com/AnKingMed/AnKing-Note-Types/tree/update-IO.  I made a new branch, made a commit to bring it current with where the PR is currently, then made another commit with my few updates that fixed it. Hopefully they make sense? You can probably make it look a bit cleaner than me, but hopefully this helps speed things up!

**Comment by RisingOrange** [comment]
I just noticed something I have to fix

**Pull request merged**